src/Surfnet/StepupRa/RaBundle/Controller/RaManagementController.php

```diff
@@ -184,10 +184,11 @@ public function createRaAction(Request $request)
         }, $raCandidate->institutions->getElements());
         $selectOptions = array_combine($options, $options);
 
-        $command                   = new AccreditCandidateCommand();
-        $command->identityId       = $identityId;
-        $command->institution      = $raCandidate->raCandidate->institution;
-        $command->raInstitution    = $this->getUser()->institution;
+        $command = new AccreditCandidateCommand();
+        $command->identityId = $identityId;
+        $command->institution = $raCandidate->raCandidate->institution;
+        $command->roleAtInstitution = new RoleAtInstitution();
```

Comment:
I would like to see the use of the constructor if that is possible in order to prevent mutability.

Reply:
Yeah I tried that on the first run, without success, as the form type should be able to set them with setters or public accessible fields.